src/main/scala/org/broadinstitute/dsde/workbench/sam/model/SamModel.scala

```diff
@@ -242,6 +242,16 @@ consistent "has a" relationship is tracked by this ticket: https://broadworkbenc
 
 @Lenses final case class GroupSyncResponse(lastSyncDate: String, email: WorkbenchEmail)
 
+final case class SamUser(id: WorkbenchUserId,
```

Comment:
Are you planning to move Pet Service Accounts to SamModel as well? What about Workbench Identities? Asking because I planned to add Azure managed identities to WB lib.

Reply:
I am not planning anything else at the moment